docs/source/models.rst

```diff
@@ -486,6 +486,28 @@ not models that implement the `scikit-learn API
 
 For more information, see :py:mod:`mlflow.lightgbm`.
 
+Spacy(``spaCy``)
```

Comment:
```suggestion
spaCy(``spaCy``)
```

Reply:
@harupy We have resorted to using a capital letter in the enumerated built-in models' list, as you can see with [here](https://mlflow.org/docs/latest/models.html#built-in-model-flavors). However, the use in the description resorts to how it's referred to in `spaCy's` documentation, though. `Fastai` is also referred to `fastai` in its documentation. So if do for `spaCy`, in this enumerated list, then we ought to do it for `fastai`, both with lower case, to be consistent. 

Or we flout that be consistent as capital for the first letter in the list, for all built-in models, to be consistent with our documentation.

Reply:
ok let's stick to the current approach (using a capital letter in the enumerated built-in models' list)